manim/mobject/geometry.py

```diff
@@ -2270,6 +2270,23 @@ def construct(self):
                 self.add(
                     line_list
                 )
+    .. manim:: FilledAngle
+        :save_last_frame:
+
+        class Fill(Scene):
```

Comment:
I think the class has to be called `FilledAngle` as in line 2273 for it to work.

Reply:
yess :)